client/lib/application/src/application-list/application-list.hbs

```diff
@@ -3,6 +3,19 @@
     <a href="/application/create" class="btn btn-default">
     <i class="fa fa-plus"></i> Create Application</a>
 </div>
+<form>
+    <input
```

Comment:
Personally I'm happy having a button that describes the action taken by the input field content, but sure I'll add a label.